src/routes/recovery/recovery.controller.ts

```diff
@@ -26,4 +35,17 @@ export class RecoveryController {
       addRecoveryModuleDto,
     });
   }
+
+  @HttpCode(200)
+  @Delete()
+  async deleteRecoveryModule(
+    @Param('chainId') chainId: string,
+    @Body(DeleteRecoveryModuleDtoValidationPipe)
+    deleteRecoveryModuleDto: DeleteRecoveryModuleDto,
```

Comment:
Since we are deleting a resource which is identified by the module address, I would say it's more semantically clear that it is part of the path: `DELETE /v1/chains/:chainId/safes/:safeAddress/recovery/:moduleAddress`

Reply:
Moved to the route in 82952c4763b1f9a0705743980b8febc963eafa67.